naucse/templates/_base_course.html

```diff
@@ -9,7 +9,7 @@
         {% if course is defined %}
             <header class="lesson-header">
                 <a href="{{ url_for('index') }}">Nauč se Python </a>
-                {% if course is defined and course.vars.get('coach-present') %}
+                {% if course.vars.get('coach-present') %}
```

Comment:
How is this relevant? Isn't it breaking anything?

Reply:
Well, it is not breaking anything, but the condition is redundant since the same one is three lines above that. But it was introduced by me in the main PR or in some follow-up...

Reply:
I see. OK. I'd rather see it in different commit but that's juts me having OCD.

Reply:
Done